packages/components/src/components/Feedback/Feedback.tsx

```diff
@@ -0,0 +1,180 @@
+};
+
+const RatingContainer = styled.div<RatingContainerProps>`
+  display: flex;
+  gap: ${ddsBaseTokens.spacing.SizesDdsSpacingLocalX05};
+  ${({ layout }) => css`
+    flex-direction: ${layout === 'horizontal' ? 'row' : 'column'};
+    align-items: ${layout === 'horizontal' ? 'center' : 'start'};
+  `}
+`;
+
+const FlexContainer = styled.div<FlexContainerProps>`
+  display: flex;
+  ${({ flexDirection }) => css`
+    flex-direction: ${flexDirection};
+    gap: ${flexDirection === 'row'
+      ? ddsBaseTokens.spacing.SizesDdsSpacingLocalX05
+      : ddsBaseTokens.spacing.SizesDdsSpacingLocalX1};
+  `};
+`;
+
+const IconLabelSpan = styled.span`
+  display: inline-flex;
+  align-items: center;
+  gap: ${ddsBaseTokens.spacing.SizesDdsSpacingLocalX05};
+`;
+
+export const Feedback = ({
+  layout = 'vertical',
+  ratingLabel,
+  positiveFeedbackLabel = 'Hva kan vi forbedre? (valgfritt)',
+  negativeFeedbackLabel = 'Hva kan vi forbedre? (valgfritt)',
+  ratingValue,
+  feedbackTextValue,
+  thumbUpTooltip = 'Bra',
+  thumbDownTooltip = 'Dårlig',
+  feedbackTextAreaExcluded = false,
+  loading = false,
+  isSubmitted,
+  onRating,
+  onFeedbackTextChange,
+  onSubmit,
+}: FeedbackProps) => {
+  const [rating, setRating] = useState<Rating | null>(null);
+  const [feedbackText, setFeedbackText] = useState<string>();
+  const [isFeedbackSubmitted, setIsFeedbackSubmitted] =
+    useState<boolean>(false);
+
+  useEffect(() => {
+    ratingValue !== undefined && setRating(ratingValue);
+  }, [ratingValue, setRating]);
+
+  useEffect(() => {
+    feedbackTextValue !== undefined && setFeedbackText(feedbackTextValue);
+  }, [feedbackTextValue, setFeedbackText]);
+
+  useEffect(() => {
+    isSubmitted !== undefined && setIsFeedbackSubmitted(isSubmitted);
+  }, [isSubmitted, setFeedbackText]);
+
+  const handleRatingChange = (newRating: Rating) => {
+    onRating && onRating(newRating);
+    onSubmit && feedbackTextAreaExcluded && onSubmit(newRating, '');
+
+    ratingValue === undefined && setRating(newRating);
+  };
+
+  const handleFeedbackTextChange = (newFeedbackText: string) => {
+    onFeedbackTextChange && onFeedbackTextChange(newFeedbackText);
+    feedbackTextValue === undefined && setFeedbackText(newFeedbackText);
+  };
+
+  const handleSubmit = () => {
+    onSubmit && onSubmit(rating!, feedbackText ?? '');
+    isSubmitted === undefined && setIsFeedbackSubmitted(true);
+  };
+
+  return (
+    <>
+      {rating === null && !isFeedbackSubmitted ? (
+        <RatingContainer layout={layout}>
+          <Label>{ratingLabel}</Label>
+          {loading ? (
+            <Spinner tooltip="Laster opp tilbakemelding ..." />
+          ) : (
+            <FlexContainer flexDirection="row">
+              <Tooltip text={thumbUpTooltip}>
+                <Button
+                  htmlProps={{
+                    style: {
+                      padding: '0px',
+                      color: ddsBaseTokens.colors.DdsColorNeutralsGray7,
+                    },
```

Comment:
Disse to knappene burde heller styles med styled-components, sånn at vi holder css-prioritetsnivået på stylingen så lavt som vi har mulighet til 😊 

```tsx
const RatingButton = styled(Button)`
  padding: 0px;
  color: ${ddsBaseTokens.colors.DdsColorNeutralsGray7}
`

<RatingButton htmlProps={{'aria-label': thumbUpTooltip}} ... />
```